scripts/startup_xline.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
```

Comment:
BTW, I think we can read environment variables to build the command for running xline, instead of using a script. That's because the script does not appear to have introduced the configuration for optional arguments such as `--jaeger-online`. After including the optional arguments, any changes to the launch parameters of xline in the future need to be synchronized to this script.
On k8s platform, FYI: https://stackoverflow.com/questions/56687542/how-to-use-environment-variable-in-kubernetes-container-command.
How do you think about it?

Reply:
Make sense. I've left an issue #551  to track this.